pycroft/lib/user_deletion.py

```diff
@@ -26,12 +26,8 @@ class ArchivableMemberInfo(Protocol):
     mem_end: datetime
 
 
-TP = t.TypeVar("TP")
-TO = t.TypeVar("TO")
-
-
 # mrh, not available in py3.10…
```

Comment:
It's Python 3.12 now so can be removed? ;)

Reply:
Not sure what this comment refers to.
I would leave it in as a "please check whether everything makes sense" marker for when work on user deletion continues.